Package.swift

```diff
@@ -1,15 +1,16 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
```

Comment:
I think it would be nice to have the Package.swift updates in a PR before this one.

Reply:
That's fine, I'll revert the changes here and open a separate PR. The change to support 12.5 is the most important right now.

Reply:
Can you update the PR description as well now?

Reply:
Done, thanks for the reminder.